src/templates/global/professional_client.html.tt

```diff
@@ -0,0 +1,40 @@
+[% PROCESS components/forms.html.tt %]
+
+[% WRAPPER fieldset legend=l('Professional Client') id='fs_professional' class="invisible" %]
+    <div class="gr-padding-10 gr-12">
+        <input id="chk_professional" type="checkbox" />
+        <label for="chk_professional">
+            [% l("I want to be treated as a professional client.") %]
+        </label>
+        <a id="professional_info_toggle" class="toggle-arrow" href="javascript:;">[% l("What is this?") %]</a>
+        <div id="professional_info" style="display: none;">
+            <p>[% l("In order for us to be able to treat you as Professional Client, as a minimum, you have to satisfy two of the following criteria:") %]</p>
+            <ul class="checked">
+                <li>[% l("You have carried out transactions, in significant size, on the relevant market that we are going to offer you, at an average frequency of 10 per quarter of the previous four quarters; and/or") %]</li>
+                <li>[% l("The size of your instrument portfolio (including Instruments &amp; Cash), with us or elsewhere, exceeds EUR500,000, or the equivalent; and/or") %]</li>
+                <li>[% l("You have worked in the financial sector for at least 1 year in a professional position that requires knowledge of the envisaged transactions.") %]</li>
+            </ul>
+            <p>[% l("By opting to be treated as a Professional Client, we shall not presume that you possess market knowledge and experience as defined by the term 'Professional Client'. We will take all reasonable steps to ensure that your request for this Categorisation meets the aforementioned criteria, including, but not limited to copies of:") %]</p>
+            <ul class="bullet">
+                <li>[% l("Statements reflecting your transactions of the previous four quarters;") %]</li>
+                <li>[% l("Portfolio held elsewhere;") %]</li>
+                <li>[% l("Proof of your employment position.") %]</li>
+            </ul>
+        </div>
+        <div id="popup" class="invisible gr-padding-20 gr-gutter">
+            <h2>[% l("Professional Clients") %]</h2>
+            <p>[% l("As a Professional Client, you will be offered a lower degree of client protection, wherein:") %]</p>
+            <ul class="bullet">
+                <li>[% l("The Company presumes that you possess the experience, knowledge and expertise to make your own investment decisions and properly assess the risks involved in relation to the requested transaction/s;") %]</li>
+                <li>[% l("The Company is neither obliged to conduct an appropriateness test, nor to provide you with any risk warnings.") %]</li>
+                <li class="invisible" id="show_financial">[% l("You will not be eligible for any compensation under the Investor Compensation Scheme.") %]</li>
```

Comment:
remove this line altogether:

<li class="invisible" id="show_financial">[% l("You will not be eligible for any compensation under the Investor Compensation Scheme.") %]</li>